buildspec.yml

```diff
@@ -0,0 +1,63 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_FULL_VERSION: '1.4.1'
+    FRAMEWORK_SHORT_VERSION: '1.4'
+    AWS_DEFAULT_REGION: 'us-west-2'
+    ECR_REPO: 'sagemaker-mxnet-serving'
+    GITHUB_REPO: 'sagemaker-mxnet-serving-container'
+    EI_ACCELERATOR_TYPE: 'ml.eia1.medium'
+    GPU_INSTANCE_TYPE: 'p2.xlarge'
+    SETUP_FILE: 'setup_cmds.sh'
+    SETUP_CMDS: '#!/bin/bash\npip install --upgrade pip\npip install -U -e .\npip install -U -e .[test]'
+
+phases:
+  pre_build:
+    commands:
+      - start-dockerd
+
+      - ACCOUNT=$(aws sts get-caller-identity --query 'Account' --output text)
+
+  build:
+    commands:
+      # run linter
+      - tox -e flake8
+
+      # run unit tests
+      - tox -e py27,py36 test/unit
+
+      # create pip archive (tar name will be something like sagemaker_mxnet_serving_container-1.0.0.tar.gz, but the Dockerfiles expect sagemaker_mxnet_serving_container.tar.gz)
+      - python3 setup.py sdist
+      - mv dist/sagemaker_mxnet_serving_container-*.tar.gz dist/sagemaker_mxnet_serving_container.tar.gz
+      - cp dist/sagemaker_mxnet_serving_container.tar.gz docker/$FRAMEWORK_FULL_VERSION/py3/sagemaker_mxnet_serving_container.tar.gz
+
+      # build images
+      - python3 scripts/build_all.py --version $FRAMEWORK_FULL_VERSION --account $ACCOUNT --repo $ECR_REPO
```

Comment:
the other PR builds add a build ID to the tag - e.g. https://github.com/aws/sagemaker-mxnet-container/blob/master/buildspec.yml#L47

Reply:
attached tag with build ID.